# Do not upstream this patch. This has been already upstreamed in
# https://github.com/triton-lang/triton/commit/de46a0ede6efe7e93c2a9ebef639e36c6177c511
# Next integration will include it and this patch should be removed then.

diff --git a/third_party/amd/python/triton_amd.cc b/third_party/amd/python/triton_amd.cc
--- a/third_party/amd/python/triton_amd.cc
+++ b/third_party/amd/python/triton_amd.cc
@@ -193,9 +193,7 @@ void init_triton_amd(py::module &&m) {
             target->createMCAsmBackend(*sti, *mri, mcOptions));
         mcStreamer.reset(target->createMCObjectStreamer(
             triple, ctx, std::move(mab), mab->createObjectWriter(svos),
-            std::move(ce), *sti, mcOptions.MCRelaxAll,
-            mcOptions.MCIncrementalLinkerCompatible,
-            /*DWARFMustBeAtTheEnd=*/false));
+            std::move(ce), *sti));
 
         std::unique_ptr<llvm::MCAsmParser> parser(
             createMCAsmParser(srcMgr, ctx, *mcStreamer, *mai));
